src/github_repo_request_local.py

```diff
@@ -86,6 +94,20 @@ def parse_args():
         default=str(Path("data/updated_local_github_df_test_count.csv")),
         help="Path to the output CSV file.",
     )
+    parser.add_argument(
+        "--ttl-file",
+        type=str,
+        default=str(Path("data/all_data.ttl")),
+        help="Path to save the Turtle (TTL) format file.",
```

Comment:
The help message is vague and clashes slightly with the default. The default includes a relative path and a filename, so does `--ttl-file` refer to a filename relative to a folder (in the default's case in `data/`)? BTW I realise that there's a similar confusion for  `--input-file`  and `--output-file` so perhaps we can address all the changes in a separate, small PR?